src/util/teamsUtil.js

```diff
@@ -1,6 +1,85 @@
 import teamsQuery from '@/graphql/query/teamsQuery.graphql';
 import leaderboardsQuery from '@/graphql/query/leaderboardsQuery.graphql';
 
+export const teamCategories = [
+	{
+		value: '',
+		label: '-- All Categories --',
+	},
+	{
+		value: 'AlumniGroups',
+		label: 'Alumni Groups',
+	},
+	{
+		value: 'Businesses',
+		label: 'Businesses',
+	},
+	{
+		value: 'BusinessesInternalGroups',
+		label: 'Business - Internal Groups',
+	},
+	{
+		value: 'Clubs',
+		label: 'Clubs',
+	},
+	{
+		value: 'CollegesUniversities',
+		label: 'Colleges/Universities',
+	},
+	{
+		value: 'CommonInterest',
+		label: 'Common Interest',
+	},
+	{
+		value: 'Events',
+		label: 'Events',
+	},
+	{
+		value: 'Families',
+		label: 'Families',
+	},
+	{
+		value: 'FieldPartnerFans',
+		label: 'Field Partner Fans',
+	},
+	{
+		value: 'Friends',
+		label: 'Friends',
+	},
+	{
+		value: 'LocalArea',
+		label: 'Local Area',
+	},
+	{
+		value: 'Memorials',
+		label: 'Memorials',
+	},
+	{
+		value: 'ReligiousCongregations',
+		label: 'Religious Congregations',
+	},
+	{
+		value: 'Schools',
+		label: 'Schools',
+	},
+	{
+		value: 'SportsGroups',
+		label: 'Sports Groups',
+	},
+	{
+		value: 'YouthGroups',
+		label: 'Youth Groups',
+	},
+	{
+		value: 'Other',
+		label: 'Other',
+	}
+];
+
+export function teamCategoryFriendlyName(categoryValue) {
+	return teamCategories.find(category => category.value === categoryValue)?.label;
```

Comment:
Do we want to make the comparison case insensitive?

Reply:
I could go either way. The Enum list in the API is cased, so this matches up with that.